src/components/TodoFilter/TodoFilter.tsx

```diff
@@ -1,34 +1,58 @@
-export const TodoFilter = () => (
-  <form className="field has-addons">
-    <p className="control">
-      <span className="select">
-        <select data-cy="statusSelect">
-          <option value="all">All</option>
-          <option value="active">Active</option>
-          <option value="completed">Completed</option>
-        </select>
-      </span>
-    </p>
+import { FC } from 'react';
 
-    <p className="control is-expanded has-icons-left has-icons-right">
-      <input
-        data-cy="searchInput"
-        type="text"
-        className="input"
-        placeholder="Search..."
-      />
-      <span className="icon is-left">
-        <i className="fas fa-magnifying-glass" />
-      </span>
+interface Props {
+  query: string,
+  setQuery: (value: string) => void,
+  setSelectStatus: (value: string) => void,
+}
 
-      <span className="icon is-right" style={{ pointerEvents: 'all' }}>
-        {/* eslint-disable-next-line jsx-a11y/control-has-associated-label */}
-        <button
-          data-cy="clearSearchButton"
-          type="button"
-          className="delete"
+export const TodoFilter: FC<Props> = ({ query, setQuery, setSelectStatus }) => {
+  const reset = () => {
+    setQuery('');
+    setSelectStatus('all');
+  };
```

Comment:
But this button shows only when you have a non-empty query. 
I think it must clear only the query.

```suggestion
  const reset = () => {
    setQuery('');
  };
```